lib/opal/cli_runners/chrome.js

```diff
@@ -0,0 +1,80 @@
+var CDP = require("chrome-remote-interface");
+var fs = require("fs");
+
+// NOTE: the code above doesn't work on Windows
+// and it doesn't support interactive stdin input (which is ok for our scenario)
+var opal_code = fs.readFileSync("/dev/stdin").toString();
+
+// Chrome can't handle huge data passed to `addScriptToEvaluateOnLoad`
+// https://groups.google.com/a/chromium.org/forum/#!topic/chromium-discuss/U5qyeX_ydBo
+// The only way is to create temporary files and pass them to chrome.
+fs.writeFileSync("/tmp/chrome-opal.js", opal_code);
+fs.writeFileSync("/tmp/chrome-opal.html", "" +
```

Comment:
That's definitely a feature, maybe then you can consider using `fs.mkdtemp`:

https://nodejs.org/dist/latest-v6.x/docs/api/fs.html#fs_fs_mkdtemp_prefix_options_callback